docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -783,55 +781,54 @@ h|Kafka property name
+For a given Kafka topic, you can use the console to check {registry} instances for value or key schemas registered to those instances that match the name of the topic. If you do not have access to any {registry} instances, or you do not have value or key schemas registered to your instances that match your topic, the console provides links to {product-registry}, where you can create instances and schemas. The console also shows you the naming format you need to use when creating a new value or key schema, so that it matches the topic.
 
 [id="proc-checking-topic-for-existing-schema-matches_{context}"]
 === Checking a topic for existing schema matches
 
 [role="_abstract"]
-The following procedure shows how to use the web console to select a Kafka topic and then check an existing {registry} instance for value or key schemas that have IDs that match the name of the topic.
+The following procedure shows how to use the {product-long-kafka} web console to select a Kafka topic and then check an existing {product-long-registry} instance for value or key schemas that have IDs that match the name of the topic.
 
 Alternatively, to learn how to create a _new_ {registry} instance with a value or key schema that matches a topic, see {base-url}{topic-config-url-kafka}#proc-creating-registry-instance-and-matching-schema-for-topic_{context}[Creating a new registry instance and matching schema for a topic].
 
 .Prerequisites
-* You're logged in to the {product-kafka} web console.
-* You've created a Kafka instance with at least one topic in {product-kafka}. To learn how to do this, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
+* You have created a Kafka instance with at least one topic in {product-kafka}. To learn how to do this, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
 * You understand how to create a {registry} instance and upload a schema to be used by client applications. To learn how to do this, see {base-url}{getting-started-url-registry}[Getting started with {product-long-registry}^].
 * You have access to at least one instance in {registry} that you can check for schemas that match your topic.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances*. Click the name of the Kafka instance that contains the topic that you want to check for matching schemas in {product-registry}.
+. In the {product-kafka} {service-url-kafka}[web console], click *Kafka Instances* and select the name of the Kafka instance that contains the topic that you want to check for matching schemas in {product-registry}.
```

Comment:
Need to include a caret here: [web console ^].

Reply:
Shoot, forgot! I've fixed all 3 instances. Thank you!